src/Network/Network/PrivateLinkService/PrivateLinkServiceProvider/ProviderConfiguration.cs

```diff
@@ -18,6 +18,7 @@ static ProviderConfiguration()
             RegisterConfiguration("Microsoft.DBforPostgreSQL/servers", "2018-06-01");
             RegisterConfiguration("Microsoft.Insights/privateLinkScopes", "2019-10-17-preview");
             RegisterConfiguration("Microsoft.Storage/storageAccounts", "2019-06-01", false);
+            RegisterConfiguration("Microsoft.StorageSync/storageSyncServices", "2020-03-01", hasConnectionsURI: true);
```

Comment:
you can remove ", hasConnectionsURI: true" since it's true by default

Reply:
I like it that way, for consistency